### DISCUSSION TITLE
Use correct font for Chinese (Simplified)

### DISCUSSION THREAD

**Discussion by kj415j45** [ppy/osu#18522] [Ideas] [answered]
As far as I known, osu!lazer use Noto (JP) as CJK font. It can be good when displaying Japanese Kanji. But it would affect Chinese Characters that used in many other place. Especially when players using Chinese as UI language.

Example (Correct glyph are labeled):
![image](https://user-images.githubusercontent.com/18349191/171537044-6ba2d5e0-9954-4a91-83b1-54152788261c.png)
![image](https://user-images.githubusercontent.com/18349191/171537932-8abdffae-653f-4585-bafc-fea652706d5e.png)
![image](https://user-images.githubusercontent.com/18349191/171538527-b5a981a3-bb51-4543-962b-660db19b4e80.png)
![image](https://user-images.githubusercontent.com/18349191/171539451-4439ecaa-703f-45bc-b881-95aee686b7d2.png)

**Comment by bdach** [comment, marked as answer]
#13843
